packages/salesforcedx-vscode-core/src/orgPicker/index.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export {
+  OrgList,
+  FileInfo,
```

Comment:
We're not using `FileInfo` outside of `orgList.ts`, lets just export the things that are used outside of `orgList.ts`.